setup.py

```diff
@@ -16,7 +16,7 @@
 import re
 import textwrap
 
-from setuptools import setup, find_packages
+from setuptools import find_packages, setup
```

Comment:
Formatting changes should be in a separate commit.

Reply:
A separate commit would fail the build if ran by itself.

Reply:
Why? Make it the first commit.